subprojects/hse-3.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-git]
+url = https://github.com/hse-project/hse.git
+revision = master
```

Comment:
revision = v3.0?

Reply:
There we no tag at the time. Is there a tag now?

Reply:
oh whoops, when this PR was created there was no branch :)

Reply:
any who, waiting for the release now